packages/manager/modules/billing/src/payment/method/add/translations/Messages_fr_FR.json

```diff
@@ -37,5 +37,6 @@
   "billing_payment_method_add_sepa_direct_debit_warning_order": "Vous pourrez alors utiliser ce moyen de paiement pour finaliser votre commande.",
   "billing_payment_method_add_sepa_direct_debit_warning_owner_identity": "Attention, le propriétaire du compte bancaire doit être identique au propriétaire du compte OVHcloud.",
   "billing_payment_method_add_sepa_direct_debit_data_processing_info": "Les données collectées sont traitées par OVHcloud en qualité de responsable de traitement afin de mettre en place votre prélèvement SEPA.",
-  "billing_payment_method_add_sepa_direct_debit_digital_signature_info": "La mise en place d'un prélèvement SEPA nécessite un mandat qu'il vous sera demandé de signer électroniquement en utilisant une solution fournie par Worldline, un prestataire habilité à délivrer les services de confiance définies par le règlement \"eDIAS\" n° 910/2014 du 23 juillet 2014. Dans ce cadre, ces données lui sont communiquées."
+  "billing_payment_method_add_sepa_direct_debit_digital_signature_info": "La mise en place d'un prélèvement SEPA nécessite un mandat qu'il vous sera demandé de signer électroniquement en utilisant une solution fournie par Worldline, un prestataire habilité à délivrer les services de confiance définies par le règlement \"eDIAS\" n° 910/2014 du 23 juillet 2014. Dans ce cadre, ces données lui sont communiquées.",
+  "billing_payment_method_add_credit_card_info_for_india": "Un paiement de 2 roupies maximum sera réalisé afin de valider le bon fonctionnement de votre carte. Il vous sera intégralement remboursé dans un délai de 48 heures."
```

Comment:
**nitpick:** `2 roupies` can be sent as param to generalise and reuse the message for other subs